src/managers/RoleManager.js

```diff
@@ -227,7 +227,7 @@ class RoleManager extends CachedManager {
    * @example
    * // Delete a role
    * guild.roles.delete('222079219327434752', 'The role needed to go')
-   *   .then(deleted => console.log(`Deleted role ${deleted.name}`))
+   *   .then(() => console.log('Deleted the role'))
```

Comment:
The role isn't undefined here. It's the role which got deleted.

Reply:
The manager doesn't return anything, the role class does.

Reply:
The method returns a Promise<void>, u sure?

Reply:
> The manager doesn't return anything, the role class does.

Oh my bad.